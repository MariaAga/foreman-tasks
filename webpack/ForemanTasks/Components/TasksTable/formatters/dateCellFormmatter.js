import React from 'react';
import LongDateTime from 'foremanReact/components/common/dates/LongDateTime';
import { translate as __ } from 'foremanReact/common/I18n';

export const dateCellFormmatter = value => (
  <LongDateTime seconds date={value} defaultValue={__('N/A')} />
);
