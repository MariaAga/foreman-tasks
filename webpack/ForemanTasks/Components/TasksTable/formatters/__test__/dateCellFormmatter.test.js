import { dateCellFormmatter } from '../dateCellFormmatter';

describe('dateCellFormmatter', () => {
  it('render', () => {
    expect(dateCellFormmatter('some-value')).toMatchSnapshot();
  });
});
