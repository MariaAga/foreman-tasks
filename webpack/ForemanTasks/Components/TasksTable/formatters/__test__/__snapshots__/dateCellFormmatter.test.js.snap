// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dateCellFormmatter render 1`] = `
<LongDateTime
  date="some-value"
  defaultValue="N/A"
  seconds={true}
/>
`;
